manager/assets/modext/widgets/system/modx.grid.manager.log.js

```diff
@@ -205,19 +211,22 @@ MODx.grid.ManagerLog = function(config) {
                     fp.filter();
                 }
             }
-        },'->',{
-            xtype: 'button'
-            ,text: _('mgrlog_clear')
-            ,scope: this
-            ,handler: function() {
-                var fp = Ext.getCmp(this.config.formpanel);
-                if (fp) {
-                    fp.clearLog();
+        },'->'];
+        if (MODx.perm.mgr_log_erase === true) {
+            tbar.push({
+                xtype: 'button'
+                , text: _('mgrlog_clear')
```

Comment:
Remove the spaces after the commas, otherwise it looks strange :)

Reply:
Good old auto-formatting. Done!